tutorial/build-run.md

```diff
@@ -34,44 +34,56 @@ To initialize configuration and a `genesis.json` file for your application and a
 
 > _*NOTE*_: If you have run the tutorial before, you can start from scratch with a `nsd unsafe-reset-all` or by deleting both of the home folders `rm -rf ~/.ns*`
 
-> _*NOTE*_: If you have the Cosmos app for ledger and you want to use it, when you create the key with `nscli keys add jack` just add `--ledger` at the end. That's all you need. When you sign, `jack` will be recognized as a Ledger key and will require a device. 
+> _*NOTE*_: If you have the Cosmos app for ledger and you want to use it, when you create the key with `nscli keys add jack` just add `--ledger` at the end. That's all you need. When you sign, `jack` will be recognized as a Ledger key and will require a device.
 
 ```bash
 # Initialize configuration files and genesis file
-nsd init --chain-id namechain
+  # moniker is the name of your node
+nsd init <moniker>
 
-# Copy the `Address` output here and save it for later use 
-# [optional] add "--ledger" at the end to use a Ledger Nano S 
+
+# Copy the `Address` output here and save it for later use
+# [optional] add "--ledger" at the end to use a Ledger Nano S
 nscli keys add jack
 
 # Copy the `Address` output here and save it for later use
 nscli keys add alice
 
 # Add both accounts, with coins to the genesis file
-nsd add-genesis-account $(nscli keys show jack -a) 1000nametoken,1000jackcoin
-nsd add-genesis-account $(nscli keys show alice -a) 1000nametoken,1000alicecoin
+nsd add-genesis-account $(nscli keys show jack -a) 1000nametoken,10000000stake
+nsd add-genesis-account $(nscli keys show alice -a) 1000nametoken,10000000stake
 
 # Configure your CLI to eliminate need for chain-id flag
 nscli config chain-id namechain
 nscli config output json
 nscli config indent true
 nscli config trust-node true
+
+nsd gentx --name <key_name>
```

Comment:
What is `key-name` here? What is this step for? If I use `jack` for example, I get:

```
ERROR: account cosmos1me3a854zyrta457gqsnudwf598rese9ps64u2k is in genesis, but it only has 10000000stake available to stake, not 100000000stake
```

Reply:
looks like you need 1 more 0 on the `nscli add-geenesis-account` command